packages/itwin/grouping-mapping-widget/src/widget/components/CustomCalculationTable.tsx

```diff
@@ -123,15 +212,17 @@ export const CustomCalculationTable = ({
   }, [getAccessToken, groupId, iModelId, mappingClient, mappingId]);
 
   return (
-    <PropertyTable
-      propertyType="Custom Calculation"
-      columnsFactory={columnsFactory}
-      data={customCalculations}
-      isLoading={isLoading}
-      onClickAdd={onClickAdd}
-      refreshProperties={refresh}
-      deleteProperty={deleteProperty}
-    />
+    <div>
+      <PropertyTable
+        propertyType="Custom Calculation"
+        columnsFactory={columnsFactory}
+        data={tableData}
+        isLoading={isLoading}
+        onClickAdd={onClickAdd}
+        refreshProperties={refresh}
+        deleteProperty={deleteProperty}
+      />
+    </div>
```

Comment:
This `div` looks redundant. Can you explain why it was added?

Reply:
This was an oversight, it was a residual "div" from a previous version.